bot/commands.php

```diff
@@ -49,3 +49,8 @@
         sendMessage($update['message']['chat']['id'], '<b>' . getTranslation('not_supported') . '</b>');
     }
 }
+// IF a Message type is private and there is no leading '/'
+else if($update['message']['chat']['type'] == 'private'){  
+    // Get Message from User sent to Bot and check what to do
+    evaluate_priv_message($update);
```

Comment:
Since this function is not available in this project this will fail if not implemented in all bots dependent on the core. `evaluate_priv_message` sounds like functionality that should live in the core, but then that can discover handlers set by individual bots. It's a bit more work but should enable way better features in the future. :-)